db-lb-app/infrastructure/openstack.yaml

```diff
@@ -8,24 +8,33 @@ imports:
 
 inputs:
 
-  region_name:
-    type: string
+  openstack_region:
+    description: ""
+    type:
+    default: { get_secret: openstack_region }
```

Comment:
Maybe we should remove this and just change `{ get_input: region_name }` below to `{get_secret: openstack_region }`?

Reply:
this way the user can override if he dont want the secret. more flexible